app/components/Widget/Voting/Voting.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { BigNumber } from 'bignumber.js'
+import {
+  Card,
+  GU,
+  IconCheck,
+  IconClose,
+  IdentityBadge,
+  ProgressBar,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { useNetwork } from '@aragon/api-react'
+import {
+  VOTING_STATUS,
+  VOTING_STATUS_ONGOING,
+  VOTING_STATUS_REJECTED,
+} from '../../../utils/constants'
+
+const Voting = ({ data }) => {
+  return (
+    <VoteContainer>
+      {data.votes.map(vote => (
+        <Vote key={vote.id}>
+          <UpperBar>
+            <App address={vote.app} />
+            <Status status={vote.status} />
+          </UpperBar>
+          <MiddleBar>
+            <Id id={vote.id}/>
+            <Description text={vote.description} />
+          </MiddleBar>
+          <LowerBar>
+            <Result yea={vote.yea} nay={vote.nay} />
+          </LowerBar>
+        </Vote>
+      ))}
+    </VoteContainer>
+  )
+}
+
+Voting.propTypes = {
+  data: PropTypes.object.isRequired,
+}
+
+const VoteContainer = styled.div`
+  display: flex;
+`
+const Vote = styled(Card)`
+  margin: ${GU}px;
+  padding: ${GU}px;
```

Comment:
I'd rather to define the all the paddings in the parent widget or here, but not half there and half here, this could make the CSS a madness to maintain